kickstarts/partials/main/disk_layout.ks.erb

```diff
@@ -2,17 +2,18 @@
 zerombr
 clearpart       --all --drives=vda
 
-part pv.1             --ondrive=vda                     --size=<%= @target == "azure" ? "11264" : "16384" %>
-part /boot            --ondrive=vda                     --size=512   --fstype=xfs
-part /var/www/miq_tmp --ondrive=vda                     --size=10240 --fstype=xfs --fsoptions="rw,noatime,nobarrier"
+part pv.1             --ondrive=vda                   --size=<%= @target == "azure" ? "21504" : "26624" %>
+part /boot            --ondrive=vda                   --size=512   --fstype=xfs
+part /var/www/miq_tmp --ondrive=vda                   --size=10240 --fstype=xfs --fsoptions="rw,noatime,nobarrier"
 
-volgroup vg_system    --pesize=4096 pv.1
-logvol /              --name=lv_os             --vgname=vg_system --size=4608 --fstype=xfs<%= " --grow" if @target == "azure" %>
-logvol /var           --name=lv_var            --vgname=vg_system --size=2048 --fstype=xfs
-logvol /var/log       --name=lv_var_log        --vgname=vg_system --size=1024 --fstype=xfs
-logvol /var/log/audit --name=lv_var_log_audit  --vgname=vg_system --size=512  --fstype=xfs
-logvol /tmp           --name=lv_tmp            --vgname=vg_system --size=1024 --fstype=xfs
-logvol /home          --name=lv_home           --vgname=vg_system --size=1024 --fstype=xfs
+volgroup vg_system           --pesize=4096 pv.1
+logvol /                     --name=lv_os             --vgname=vg_system --size=4608  --fstype=xfs<%= " --grow" if @target == "azure" %>
+logvol /var                  --name=lv_var            --vgname=vg_system --size=2048  --fstype=xfs
+logvol /var/log              --name=lv_var_log        --vgname=vg_system --size=1024  --fstype=xfs
+logvol /var/log/audit        --name=lv_var_log_audit  --vgname=vg_system --size=512   --fstype=xfs
+logvol /var/www/miq/vmdb_log --name=lv_log            --vgname=vg_system --size=10240 --fstype=xfs
```

Comment:
To @Fryguy 's point in https://github.com/ManageIQ/manageiq-appliance-build/pull/159/files#r79827931, this should just be `/var/www/miq/vmdb/log`